raiden/network/proxies/token_network.py

```diff
@@ -2213,47 +2239,66 @@ def settle(
             # gas will stop us from sending a transaction that will fail
             pass
 
-        with self.channel_operations_lock[partner]:
-            error_prefix = "Call to settle will fail"
-            gas_limit = self.proxy.estimate_gas(
-                checking_block, "settleChannel", channel_identifier=channel_identifier, **kwargs
-            )
+        if self_or_lc_address == self.node_address:
+            # is not lc
+            with self.channel_operations_lock[partner]:
+                error_prefix = "Call to settle will fail"
```

Comment:
we can do either uppercase or lowercase on prefixes, but let's be consistent on this